extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/TLSConfig.java

```diff
@@ -3,36 +3,43 @@
 import java.util.Optional;
 
 import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
 
 @ConfigGroup
 public class TLSConfig {
     /**
-     * @return the cert authority certificate contents.
+     * The cert authority certificate contents.
      */
+    @ConfigItem
     Optional<String> caCertificate;
 
     /**
-     * @return the certificate contents.
+     * The certificate contents.
```

Comment:
Does "contents" means **text** of certificate (2k bytes of base64 starting with "-----BEGIN CERTIFICATE-----") ? If so, how do we paste that into application.properties?

(this was the most important thing I wanted to see in the guide)